scripts/publish-staged

```diff
@@ -33,6 +33,7 @@ usage() {
 	echo "Options"
 	echo "  -d  true|false  Dryrun (default is true)"
 	echo "  -p  PROFILE     AWS CLI Profile (default is none)"
+	echo "  -o  PROFILE     AWS CLI Profile for the bucket to push to (defaults to the profile specified with -p)"
```

Comment:
Its necessary for pushing to the bucket in cn-north-1. That's because the China regions are required to have separate accounts- so the argument for the `-p` flag is the profile used to read from the staging bucket, and the `-o` (output) flag is the profile that the binary is pushed to.
If `-o` is not specified then it defaults to the value of `-p`: this is because when we are releasing binaries to regions that are not partitioned (ie not china), then only 1 AWS profile is needed- so the script allows you to specify only one account just like before.